pkg/kubectl/cmd/config/current_context_test.go

```diff
@@ -66,6 +66,7 @@ func (test currentContextTest) run(t *testing.T) {
 
 	pathOptions := NewDefaultPathOptions()
 	pathOptions.GlobalFile = fakeKubeFile.Name()
+	pathOptions.EnvVar = ""
```

Comment:
> or set the EnvVar name to something other than KUBECONFIG. Mostly, I want to continue testing the fallback behavior through an empty envvar, rather than skipping the envvar check

That's not what these tests do.  Those tests are in a different location.  This is just trying to make sure they can output the current-context correctly.